<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1013</width>
         <height>1762</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>130</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>New_Sinhgad_Logo_2013_300-removebg-preview.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>112</width>
               <height>150</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>logo.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>sih_logo.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>aquaprobe_logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>450</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Introducing AquaProbe – Empowering Water Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>850</width>
               <height>270</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;- AquaProbe is a desktop application for analysis of pumping test data.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;- The application was developed under the guidance of Central Ground Water Board (CGWB) as a part of Smart India Hackathon 2022. &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;- The application facilitates the estimation of aquifer parameters such as Transmissivity, Storativity and Specific Yield. &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;- It facilities for analysis of pumping test data using Cooper Jacob, Theis and Theis Recovery method.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;- AquaProbe is a freeware. However please read the detailed terms of use.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;- In scientific publications/reports, use of this application may be cited as follows - Gidwani, S., &amp;amp; Ghatage, D. (2024). AquaProbe: A Desktop Application for Water Analysis. Developed under SIH 2022. Central Ground Water Board (CGWB), Ministry of Jal Shakti, Govt of India.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;By using the application, you acknowledge and agree to abide by the terms of use.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="review_terms_button">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>View Terms of Use</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="welltable_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Well Table</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createwell_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Create Well</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>450</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="carousel_images_label">
                <property name="minimumSize">
                 <size>
                  <width>800</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>400</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 1px solid;</string>
                </property>
                <property name="text">
                 <string>Carousel</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>850</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>850</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Cooper Jacob</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="styleSheet">
               <string notr="true">QScrollArea {
    border: none;
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>809</width>
                 <height>436</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>The Cooper and Jacob (1946) solution (sometimes called Jacob's modified nonequilibrium method) is a late-time approximation derived from the Theis type-curve method.


Assumptions:

 - Aquifer has infinite areal extent
 - Aquifer is homogeneous, isotropic and of uniform thickness
 - Control well is fully penetrating
 - Flow to control well is horizontal - Aquifer is nonleaky confined
 - Flow is unsteady
 - Water is released instantaneously from storage with decline of hydraulic head
 - Diameter of pumping well is very small so that storage in the well can be neglected
 - Values of 𝑢 are small (i.e., 𝑟 is small and 𝑡 is large)

Data Requirements:

 - 𝑄, pumping rate [m³/day]
 - distance from well [m]
observation well measurements (time and displacement)

 - Estimated Parameters:

 - S, Storativity
 - T, Transmissivity [m²/day]
 - Mean fitting error</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Theis</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QScrollArea" name="scrollArea_3">
              <property name="styleSheet">
               <string notr="true">QScrollArea {
    border: none;
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>826</width>
                 <height>385</height>
                </rect>
               </property>
               <widget class="QLabel" name="label_10">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>808</width>
                  <height>320</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>The Theis (1935) solution is useful for determining the transmissivity of nonleaky confined aquifers from pumping tests.


Assumptions:

 - Aquifer has infinite areal extent
 - Aquifer is homogeneous, isotropic and of uniform thickness
 - Control well is fully penetrating
 - Flow to control well is horizontal
 - Aquifer is nonleaky confined
 - Flow is unsteady
 - Water is released instantaneously from storage with decline of hydraulic head
 - Diameter of pumping well is very small so that storage in the well can be neglected

Data Requirements:

 - 𝑄, pumping rate [m³/day]
 - distance from well [m]
 - observation well measurements (time and displacement)

Estimated Parameters:

 - S, Storativity
 - T, Transmissivity [m²/day]
 - Mean fitting error


</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Theis Recovery</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QScrollArea" name="scrollArea_4">
              <property name="styleSheet">
               <string notr="true">QScrollArea {
    border: none;
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_4">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>809</width>
                 <height>421</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>The Theis (1935) solution is useful for determining the transmissivity of nonleaky confined aquifers from recovery tests.


Assumptions:

 - Aquifer has infinite areal extent
 - Aquifer is homogeneous, isotropic and of uniform thickness
 - Control well is fully penetrating
 - Flow to control well is horizontal
 - Aquifer is nonleaky confined
 - Flow is unsteady
 - Water is released instantaneously from storage with decline of hydraulic head
 - Diameter of pumping well is very small so that storage in the well can be neglected
 - Values of 𝑢 are small (i.e., 𝑟 is small and 𝑡 is large)

Data Requirements:

 - 𝑄, pumping rate [m³/day]
 - distance from well [m]
 - observation well measurements (time and displacement)

Estimated Parameters:

 - T, Transmissivity [m²/day]
 - 𝑆/𝑆′ (ratio of storativity during pumping to storativity during recovery)</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWellTable">
    <property name="title">
     <string>Well Table</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About Us</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuWellTable"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
